### PR TITLE
Add custom themed Raspibolt backup card

### PR DESCRIPTION
**Pull request by sutterseba**
### What

@VajraOfIndra suggested on Reddit that a printable backup card [like the one from Shift Crypto](https://shiftcrypto.ch/backupcard/backupcard_print.pdf), but in a Raspibolt theme, would be neat. 

I'm not that skilled with illustration software, but I gave it my best... :D

The original is licensed under CC BY-SA 4.0, allowing to use it as a template.

### How

Added `raspibolt-backup-card.pdf` to `resources` and edited section (and link) in the Lightning client guide that refers to the custom backup card.

The link will not work until merged, but [here is the same link](https://github.com/sutterseba/raspibolt/blob/custom-backup-card/resources/raspibolt-backup-card.pdf) on my fork. I prefer this over a raw link to allow users having a look at it first before downloading.

If you like, I can provide the original Affinity Designer project file if someone is interested or wants to improve on it.

### CODE REVIEW

**Comment by VajraOfIndra** [comment]
That's great! Thanks a lot for the initiative! :heart: 

I see that you're acknowledging the ShiftCrypto card as model on your card, perfect! :+1: 

@Stadicus, wdyt? It'd be nice to keep a link to the ShiftCrypto one though.. see proposal below as one solution.

I know someone else working on a design. If more than one design are proposed, I suggest we create a repo for them where we drop the different models (we could add the ShiftCrypto one also) and in the guide we link to that repo for download. We could also create a preview of all the models in the readme file of the repo (not sure if it works with PDF but definitely works nice with pics, see e.g. [here](https://github.com/VajraOfIndra/awesome-bitcoin-logos))

```
raspibolt/backup-cards
|_png
   |_raspibolt3-(1).png
   |_raspibolt3-(2).png
   |_raspibolt2.png
   |_shiftcrypto.png
|_LICENSE
|_README.md
raspibolt/raspibolt
|_ ...
raspibolt/v2.raspibolt.org 
|_ ...
```

**Comment by Ralph-Naders-ghost** [comment]
Made 2 cards.  One for U.S. paper sizes (8 1/2 X 11) and one for European paper sizes (210mm X 297mm)

[US Lightning V3 backup card.pdf](https://github.com/raspibolt/raspibolt/files/9387922/US.Lightning.V3.backup.card.pdf)
[Euro Raspibolt V3 backup card.pdf](https://github.com/raspibolt/raspibolt/files/9387924/Euro.Raspibolt.V3.backup.card.pdf)

**Comment by sutterseba** [comment]
@Ralph-Naders-ghost Leaving room for the user passwords is a great idea! I added that to my version as well.

**Comment by Stadicus** [comment]
> Made 2 cards. One for U.S. paper sizes (8 1/2 X 11) and one for European paper sizes (210mm X 297mm)

Wow, each card is 20 MB... I think that can be optimized... ;-)

**Comment by Stadicus** [comment]
> The original is licensed under CC BY-SA 4.0, allowing to use it as a template.

Haha, great job! This backup card came full circle, as I'm the original author of the BitBox backup card as well... :man_astronaut::+1:

File: guide/lightning/lightning-client.md [hunk 1]
```diff
@@ -278,8 +278,10 @@ The current state of your channels, however, cannot be recreated from this seed.
 For this, the Static Channel Backup stored at `/data/lnd-backup/channel.backup` is updated continuously.
 
 🚨 This information must be kept secret at all times.
+
 * **Write these 24 words down manually on a piece of paper and store it in a safe place.**
-You can use a simple piece of paper, write them on a proper [backup card](https://shiftcrypto.ch/backupcard/backupcard_print.pdf){:target="_blank"}), or even stamp the seed words into metal (see this [DIY guide](https://www.econoalchemist.com/post/backup){:target="_blank"}).
+
+You can use a simple piece of paper, write them on the custom themed [Raspibolt backup card](https://github.com/raspibolt/raspibolt/blob/master/resources/raspibolt-backup-card.pdf){:target="_blank"}, or even stamp the seed words into metal (see this [DIY guide](https://www.econoalchemist.com/post/backup){:target="_blank"}).
```
Comment (Stadicus) [inline guide/lightning/lightning-client.md:284]: s/Raspibolt/RaspiBolt

**Review by Stadicus — APPROVED** [review]
I'm fine with removing my BitBox backup card, and replacing it with the RaspiBolt card. No hard feelings :)

**Comment by sutterseba** [comment]
Cool! :+1:

I think we can go to master then? If @Ralph-Naders-ghost wants to update (the size of) his version and/or others want to create backup cards as well we can always go for an extra repo but I think the single card is the easiest for now.

I put the project files [on my fork](https://github.com/sutterseba/raspibolt/tree/project-files/resources) if anyone wants to use them.

**Pull request merged**